src/constants.js

```diff
@@ -1,11 +1,16 @@
+const packageInfos = require('../package.json')
 const ADD_COZY_METADATA = true
-const APP_NAME = 'konnector-google'
+const APP_NAME = packageInfos.name
+const APP_VERSION = packageInfos.version
 const DOCTYPE_CONTACTS = 'io.cozy.contacts'
 const DOCTYPE_CONTACTS_ACCOUNT = 'io.cozy.contacts.accounts'
+const DOCTYPE_CONTACTS_VERSION = 2
```

Comment:
Are we sure about (int) 2? Can we have 2.1 one day?

Reply:
I think 2 is correct — [the documentation](https://github.com/cozy/cozy-doctypes/blob/master/docs/README.md#document-metadata) says so, and I believe every version upgrade indicated a breaking change in the doctype, so I don't think there will ever be intermediate versions.

(Naturally now that I've said this it's bound to happen some day, but oh well…)